CHANGES.rst

```diff
@@ -1,6 +1,33 @@
 Changes
 =======
 
+Version 0.5.0 (2019-04-19)
+--------------------------
+
+- Improves command line tool documentation by grouping of commands and
+  command examples.
+- Centralises the way workflow name is passed to CLI commands.
```

Comment:
This might sound too technical and may not have too much value to an end user, I would either place it last or remove it, WDYT?